.github/workflows/configuration-documentation.yml

```diff
@@ -32,4 +32,4 @@ jobs:
         uses: stefanzweifel/git-auto-commit-action@v4
         with:
           commit_message: Configuration documentation updated
-          file_pattern: doc/CONFIGURATION.json docs/modules/ROOT/pages/ios_mdm_tables.adoc
+          file_pattern: doc/CONFIGURATION.json doc/antora/pages/ios_mdm_tables.adoc
```

Comment:
Please generate on 2 places:

- doc/CONFIGURATION.adoc
- doc/antora/pages/mdm_tables.adoc (wie MMattel will…)

Reply:
A copy (double generation) of `mdm_tables.adoc` named `doc/CONFIGURATION.adoc` does not make any sense especially because you want to combine several markdown files into `doc/CONFIGURATION.adoc` for ios-app internal documentation purposes, see your comment: https://github.com/owncloud/ios-app/pull/1094#issuecomment-1020176889. Therefore I guess this was just a mistake 😄

Reply:
Nothing gets combined. Other `*.md`,s will just be deleted. I need the MDM content accessible in `doc/CONFIGURATION.adoc`. If you want to have the content in `doc/antora/pages/`, @hosy will add a copy for your purpose there. up2u…